### DISCUSSION TITLE
[multi session] asking for QR CODE again

### DISCUSSION THREAD

**Discussion by rogeriopavan** [dimaskiddo/go-whatsapp-multidevice-rest#51] [Q&A] [answered]
Hello!
When the api is restarted, it restoring the sessions file, but the api generates the QRCODE again. 
Is this working as expected? or am I doing something wrong?  

I see that the startup.go restore the sessions, but i think

**Comment by ibnux** [comment, marked as answer]
it will always ask for QRcode, if the number you register it different with number you scan it

it must use phone number to Auth and get JWT
https://github.com/dimaskiddo/go-whatsapp-multidevice-rest/wiki/Endpoint#auth

**Reply by rogeriopavan** [reply]
Nice! It works.
